com.woltlab.wcf/option.xml

```diff
@@ -1701,5 +1693,8 @@ DESC:wcf.global.sortOrder.descending</selectoptions>
 	</import>
 	<delete>
 		<option name="module_master_password"/>
+		<option name="blacklist_ip_addresses"/>
+		<option name="blacklist_user_agents"/>
+		<!-- <optioncategory name="security.blacklist.custom"/> -->
```

Comment:
Not sure about this one. It might be in use by plugins.

Reply:
An empty category doesn't hurt, we can deprecate it, but keep it for the 5.x series around.

Reply:
If it is not removed, I would this commented.out line.